### DISCUSSION TITLE
Word correction in documentation

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23360] [New to GitHub]
<p>Hello,</p>
<p>I am reading the following documentation in Brazilian Portuguese <a href="https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/upgrading-from-dependabotcom-to-github-native-dependabot" rel="noopener nofollow ugc">Upgrading from Dependabot.com to GitHub-native Dependabot</a>.</p>
<p>However, in the text that says "For more information about the upgrade pull request, see “Upgrading to GitHub-native Dependabot” in the Portuguese version Brazil has the word “para” with two letters “p”.</p>
<p>It looks like this: Para obter mais informações sobre o pull request de atualização, consulte "Atualizando <strong>ppara</strong> o Dependabot nativo do GitHub.</p>
<p>So the correct phrase would be → Para obter mais informações sobre o pull request de atualização, consulte "Atualizando <strong>para</strong> o Dependabot nativo do GitHub.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/1/01bba1f88af1bcf3a9bcc75b9e3919d4a886a6bf.png" data-download-href="/uploads/short-url/fktG1hpKrrr86Td8a79CnnFux1.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/87830226/181076533-46c9b109-91e4-4a39-9f50-a4a501e09e92.png" alt="image" data-base62-sha1="fktG1hpKrrr86Td8a79CnnFux1" width="690" height="134" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/1/01bba1f88af1bcf3a9bcc75b9e3919d4a886a6bf_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">731×143 7.67 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jsoref** [comment]
<aside class="onebox githubblob" data-onebox-src="https://github.com/github/docs/blob/c59ec33146eb990443f18f5a979fe1243d55252a/CONTRIBUTING.md#earth_asia-translations">
  <header class="source">

      <a href="https://github.com/github/docs/blob/c59ec33146eb990443f18f5a979fe1243d55252a/CONTRIBUTING.md#earth_asia-translations" target="_blank" rel="noopener">github.com</a>
  </header>

  <article class="onebox-body">
    <h4><a href="https://github.com/github/docs/blob/c59ec33146eb990443f18f5a979fe1243d55252a/CONTRIBUTING.md#earth_asia-translations" target="_blank" rel="noopener">github/docs/blob/c59ec33146eb990443f18f5a979fe1243d55252a/CONTRIBUTING.md#earth_asia-translations</a></h4>


    <pre><code class="lang-md"># Contributing to this repository &lt;!-- omit in toc --&gt;

## Getting started &lt;!-- omit in toc --&gt;

Before you begin:
- This site is powered by Node.js. Check to see if you're on the [version of node we support](contributing/development.md).
- Have you read the [code of conduct](CODE_OF_CONDUCT.md)?
- Check out the [existing issues](https://github.com/github/docs/issues) &amp; see if we [accept contributions](#types-of-contributions-memo) for your type of issue.

### Use the 'make a contribution' button

&lt;img src="./assets/images/contribution_cta.png" width="400"&gt;

Navigating a new codebase can be challenging, so we're making that a little easier. As you're using docs.github.com, you may come across an article that you want to make an update to. You can click on the **make a contribution** button right on that article, which will take you to the file in this repo where you'll make your changes.

Before you make your changes, check to see if an [issue exists](https://github.com/github/docs/issues/) already for the change you want to make.

### Don't see your issue? Open one

If you spot something new, open an issue using a [template](https://github.com/github/docs/issues/new/choose). We'll use the issue to have a conversation about the problem you want to fix.
</code></pre>


  This file has been truncated. <a href="https://github.com/github/docs/blob/c59ec33146eb990443f18f5a979fe1243d55252a/CONTRIBUTING.md#earth_asia-translations" target="_blank" rel="noopener">show original</a>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by alebourne** [comment, marked as answer]
<p><a class="mention" href="/georgebias">@georgebias</a> Muito obrigada!</p>
<p>I was able to easily make this correction. It should appear fixed on the pt-BR site in a few days.</p>
<p>Also, if you find more grammar or spelling errors you can report them to our Support team by clicking the Contact Support button on the bottom right corner in the documentation site <a href="http://docs.github.com" rel="noopener nofollow ugc">docs.github.com</a></p>
<p>Ciao!</p>
